corona-warn-app/cwa-app-android
# Contact Journal - Keyboard stays open (EXPOSUREAPP-6702)

This PR addresses Exposureapp-6702 (Keyboard stays open on person or location deletion).

**How to test:**
- Open "Personen bearbeiten" or "Orte bearbeiten" screen
- press "Edit" on person or location
- delete the person or location

Expected result:
Keyboard is closed on person or location delete
